recipes/openforcefield/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "openforcefield" %}
+{% set version = "0.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openforcefield/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: cd4ca0ded60ebd97120523b85c2aa6e0ddbc400193e6bbf49aa52b82ccf9e026
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python >=3.6
+    - numpy
+    - smirnoff99frosst
+    - openforcefields
+    - openmm
+    - networkx
+    - rdkit
+    - ambertools >=20
+    # Serialization: Should these be optional?
+    - toml
+    - bson
+    - msgpack-python
+    - xmltodict
+    - pyyaml
+    - cairo >=1.16
+    # Development? Needed?
+    - packaging
+    - pytest
+    - requests
+    - notebook
+    - nglview
```

Comment:
I think these are all development, testing, used in examples, or otherwise dependencies I'd argue should be optional. The examples might be tricky to handle since we probably want users to be able to do `conda install openforcefield -c conda-forge` and run the examples out of the box, although this could be potentially be changed to require the user to do a local install (since, if running examples, the user would probably already have it cloned). Not sure.

Reply:
We could have a multi-output recipe that packages the examples to `openforcefield-examples` and adds the extra dependencies (although probably to be handled in the corresponding feedstock, not now).